src/features/l2ps-messaging/L2PSMessagingServer.ts

```
@@ -0,0 +1,454 @@
+                this.sendError(ws, "INVALID_PROOF", "Signature verification failed")
+                return
+            }
+        } catch (error) {
+            this.sendError(ws, "INVALID_PROOF", `Proof verification error: ${error}`)
+            return
+        }
+
+        // Remove old connection if re-registering
+        const existing = this.peers.get(publicKey)
+        if (existing) {
+            try { (existing.ws as ServerWebSocket<WSData>).close() } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)